tailwind.config.js

```diff
@@ -0,0 +1,34 @@
+const production = !process.env.ROLLUP_WATCH;
+
+const svelteExtractor = (content) => {
+  const regExp = new RegExp(/[A-Za-z0-9-_:/]+/g);
+  const matchedTokens = [];
+  let match = regExp.exec(content);
+
+  while (match) {
+    if (match[0].startsWith("class")) {
+      matchedTokens.push(match[0].substring(6));
+    } else {
+      matchedTokens.push(match[0]);
+    }
+    match = regExp.exec(content);
+  }
+  return matchedTokens;
+};
```

Comment:
Would you mind testing this regex on this page? If this works it will avoid another loop and may be faster to parse...

https://github.com/sarioglu/svelte-tailwindcss-template/blob/master/tailwind.config.js

Reply:
This extractor looks great, however I feel like `/(?:class:)*([\w\d-/:%.]+)/` is missing checking for underscores and would like to add `_` to the regex. What do you think @robhudson?

Reply:
The `\w` matches any word character (equal to [a-zA-Z0-9_]) so it includes the underscore.

Reply:
You're right, thank you for clarifying! Tested and everything worked. I've updated the code with this new extractor.